eng/ILLink.Substitutions.Resources.template

```diff
@@ -1,10 +1,16 @@
 <linker>
   <!-- System.Resources.UseSystemResourceKeys removes resource strings and instead uses the resource key as the exception message -->
   <assembly fullname="{AssemblyName}" feature="System.Resources.UseSystemResourceKeys" featurevalue="true">
-    <!-- System.Resources.UseSystemResourceKeys removes resource strings and instead uses the resource key as the exception message -->
     <resource name="{StringResourcesName}.resources" action="remove" />
     <type fullname="System.SR">
       <method signature="System.Boolean UsingResourceKeys()" body="stub" value="true" />
+      <method signature="System.Boolean GetUsingResourceKeysSwitchValue()" body="stub" value="true" />
```

Comment:
This is [#95948 ](https://github.com/dotnet/runtime/pull/95948#issuecomment-1878227196) as a proof that it works.